sdk/tests/conformance2/extensions/webgl-shader-pixel-local-storage.html

```diff
@@ -0,0 +1,408 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>WebGL WEBGL_shader_pixel_local_storage Conformance Tests</title>
+<link rel="stylesheet" href="../../resources/js-test-style.css"/>
+<script src="../../js/desktop-gl-constants.js"></script>
+<script src="../../js/js-test-pre.js"></script>
+<script src="../../js/webgl-test-utils.js"></script>
+<script src="../../js/tests/compositing-test.js"></script>
+<script src="../../js/tests/invalid-vertex-attrib-test.js"></script>
+</head>
+<body>
+<div id="description"></div>
+<canvas id="canvas" width="128" height="128"> </canvas>
+<div id="console"></div>
+<script>
+"use strict";
+description("This test verifies the functionality of the WEBGL_shader_pixel_local_storage " +
+            "extension, if it is available.");
+
+const wtu = WebGLTestUtils;
+const canvas = document.getElementById("canvas");
+const gl = wtu.create3DContext(canvas, {alpha: false}, 2);
```

Comment:
Could the test be repeated for the default (`alpha: true`) case? No-alpha composition may use different code paths.

Reply:
Create a regular canvas from JS, get an alpha-enabled context, run tests, delete that canvas, repeat with alpha disabled.

It's a good idea to check PLS operations with `OffscreenCanvas` but it would better be a separate test.

Reply:
Done! Can't quite delete it yet though because it needs to go to sleep for compositing.

This test also captures PLS being active on two contexts at once.

OffscreenCanvas might be a good candidate for some of the future tests I can import from ANGLE.

Does this look good to land now?